taxonomies/packaging_materials.txt

```diff
@@ -163,7 +163,7 @@ pt:PET opaco
 <en:Plastic
 en:HDPE - High-density polyethylene, High-density polyethylene
 xx:HDPE, HD-PE, RC 2, RC 02, RC2, RC02, HDPE 2, 2 HDPE, HDPE 02, 02 HDPE, PEHD, 2 PEHD, PEHD 2, 02 PEHD, PEHD 02, PE-HD, ♴
-bg:HDPE - Полиетилен висока плътност
+bg:HDPE - Полиетилен с висока плътност
```

Comment:
Maybe add the form with "c" as a variant (to capture both) ? (same for LDPE)

Reply:
I checked and officially they write it with the "c" but if its necesseray to have both variants I will edit the file.